src/pages/memories/MemoriesComponent.js

```diff
@@ -0,0 +1,59 @@
+import Header from "../../components/header/Header";
+import Footer from "../../components/footer/Footer";
+import { memories } from "../../portfolio";
```

Comment:
```suggestion
```

Reply:
It should to be there. Since, all the information is import from `portfolio.js` which is not the case.